src/ingestors/fxhash/index.ts

```diff
@@ -0,0 +1,127 @@
+import { MintIngestor, MintIngestorResources } from '../../lib/types/mint-ingestor';
+import { MintIngestionErrorName, MintIngestorError } from '../../lib/types/mint-ingestor-error';
+import { MintInstructionType, MintTemplate } from '../../lib/types/mint-template';
+import { MintTemplateBuilder } from '../../lib/builder/mint-template-builder';
+import { FXHASH_BASE_FIXED_PRICE_ABI, FXHASH_BASE_FRAME_ABI } from './abi';
+import { getFxHashMintPriceInEth } from './onchain-metadata';
+
+const BASE_FRAME_CONTRACT_ADDRESS = '0x6e625892C739bFD960671Db5544E260757480725';
+const BASE_FIXED_PRICE_CONTRACT_ADDRESS = '0x4bDcaC532143d8d35ed759189EE22E3704580b9D';
+
+export class FxHashIngestor implements MintIngestor {
+  async supportsUrl(url: string): Promise<boolean> {
+    return new URL(url).hostname === 'www.fxhash.xyz' || new URL(url).hostname === 'fxhash.xyz';
+  }
+
+  async createMintTemplateForUrl(url: string, resources: MintIngestorResources): Promise<MintTemplate> {
+    const isCompatible = await this.supportsUrl(url);
+    if (!isCompatible) {
+      throw new MintIngestorError(MintIngestionErrorName.IncompatibleUrl, 'Incompatible URL');
+    }
+
+    const mintBuilder = new MintTemplateBuilder()
+      .setOriginalUrl(url)
+      .setMintInstructionType(MintInstructionType.EVM_MINT)
+      .setPartnerName('FxHash');
```

Comment:
fwict from their brand they tend to style it `fxhash` — I've created that partner on our end now so it'll match on Floor with the updated casing 😊

Reply:
Done!